frontend/src/modules/member/components/view/_aside/_aside-enriched.vue

```diff
@@ -1,12 +1,22 @@
 <template>
   <div>
     <!-- Education -->
-    <div
-      v-if="member.attributes.education"
-      class="font-medium text-black mb-4 mt-10"
-    >
-      Education
+    <div v-if="member.attributes.education" class="flex items-center mb-4 mt-10">
+      <div
+        class="font-medium text-black mr-2"
+      >
+        Education
```

Comment:
You're only adding this to "Education" in this file. There's also Certifications and Awards

Reply:
sorted this out, didnt know about those 2